uwblueprint/social-diversity-for-children
# Setup Authentication and Boilerplate Sign Up Page

### Notion ticket link



[Set up Authentication](https://www.notion.so/uwblueprintexecs/Set-up-Authentication-217f2c263d69405fa82fc3975fe15686)



### Implementation description
- Added dependencies for `nextauthjs`
- Set up boilerplate authentication flow 
- Did not add any custom callbacks yet as that will need to be done after the schemas have been added 
- Did not create a custom email template - to be done in a separate ticket 
- Did not create a verify page - to be done in a separate ticket



### Steps to test

1. Spin up the app and try entering in your email to ensure that the login flow is working as expected. Note that you will need to verify your email in AWS as we are still in sandbox mode.



### What should reviewers focus on?
-

### Checklist
